model/src/w3profsmd_pdlib.F90

```diff
@@ -5585,1055 +5529,648 @@ SUBROUTINE PDLIB_JACOBI_GAUSS_SEIDEL_BLOCK(IMOD, FACX, FACY, DTG, VGX, VGY)
     USE yowfunction, only : pdlib_abort
     USE yowNodepool, only: np_global
     USE W3DISPMD, only : WAVNU_LOCAL
-
+    USE W3ADATMD, ONLY: U10, U10D
+#ifdef W3_ST4
+    USE W3SRC4MD, only: W3SPR4
+#endif
+#ifdef W3_REF1
+      USE W3GDATMD, only: REFPARS
+#endif
+#ifdef W3_MEMCHECK
+    USE MallocInfo_m
+#endif
```

Comment:
@aronroland it was mentioned by @DeniseWorthen that some of the `call print_memcheck` lines had been taken out, and the old `#ifdef W3_MEMCHECK` inserted inserted back in for the PR.  Was this on accident? Will you please replace with the current `call print_memcheck` statements?

Reply:
Thanks for catching this @DeniseWorthen!

Reply:
@MatthewMasarik-NOAA can u tell me which ones, I just looked checked the code and deleted the #ifdef W3_MEMCHECK part but i did not see the print_memcheck missing somewhere ...?